bzt/utils.py

```diff
@@ -1286,7 +1286,7 @@ def add_imports(self):
         return imports
 
     @abstractmethod
-    def build_source_code(self):
+    def build_source_code(self, execution=None):
```

Comment:
Why this is changed?

Reply:
I revert that change on this commit (Revert the file to master version, cleaning the PR to allowing only pure reporting changes) https://github.com/Blazemeter/taurus/pull/838/commits/7a43c3c9366f61c88da8eceebf46f4dc4b9f998a

And create a PR related to the refactoring to outside this PR https://github.com/Blazemeter/taurus/pull/846

The argument you mention is something that no longer goes in this PR (and not included in the 846 PR, in that only changes the structure to minimize use of variables)